ydb/docs/en/core/integrations/import-jdbc.md

```diff
@@ -0,0 +1,256 @@
+        <key-column>OWNER</key-column>
+        <key-column>TABLE_NAME</key-column>
+    </table-ref>
+</ydb-importer>
+```
+
+## Import tool limitations {#limitations}
+
+Right now the tested data sources include the following:
+
+* [PostgreSQL](https://www.postgresql.org/),
+* [MySQL](https://www.mysql.com/),
+* [Oracle Database](https://www.oracle.com/database/),
+* [Microsoft SQL Server](https://www.microsoft.com/sql-server/),
+* [IBM Db2](https://www.ibm.com/products/db2),
+* [IBM Informix](https://www.ibm.com/products/informix).
+
+Other data sources will probably work, too, as the tool uses the generic JDBC APIs to retrieve data and metadata.
+
+Secondary indexes are not imported.
+
+Some data types and table structures are known to be unsupported:
+
+* the embedded tables for Oracle Database,
+* spatial data type for Microsoft SQL Server,
+* object data types for Informix.
+
+### Handling tables without the primary key {#nopk}
+
+Each YDB table must have a primary key. If a primary key (or at least a unique index) is defined on the source table, the tool creates the primary key for the target YDB table with the columns and order defined by the original primary key. When having multiple unique indexes defined on the source table, the tool prefers the index with the smallest number of columns.
+
+In case the source table has no primary key and no unique indexes, primary key columns can be configured using the import settings in the configuration file, as a sequence of `key-column` elements in the `table-ref` section (see below the example in the description of the configuration file format).
+
+When there is no primary key defined anywhere, the tool automatically adds the column `ydb_synth_key` to the target table, and uses it as the primary key. The values of the synthetic primary key are computed as "SHA-256" hash code over the values of all columns of the row, except the columns of the BLOB type.
+
+If the input table contains several rows with completely identical values, the destination table will have only one row per each set of duplicate input rows. This also means that there the tool cannot import the table in which all columns are of BLOB type.
+
+### Importing large objects (BLOB, XML) {#blob}
+
+For each BLOB field in the source database the import tool creates an additional YDB target table (BLOB supplemental table), having the following structure:
+
+```sql
+CREATE TABLE `blob_table`(
+    `id` Int64,
+    `pos` Int32,
+    `val` String,
+    PRIMARY KEY(`id`, `pos`)
+)
+```
+
+The name of the BLOB supplemental table is generated based on the `table-options` / `blob-name-format` setting in the configuration file.
+
+Each BLOB field value is saved as a sequence if records in the BLOB supplemental table in YQB. Actual data is stored in the `val` field, storing no more than 64K bytes. The order of blocks stored is defined by the values of the `pos` column, containing the integer values `1..N`.
```

Comment:
So it sounds like there's never pos=0, or is it a special value for something?

Reply:
> So it sounds like there's never pos=0, or is it a special value for something?

Yes, zero is never used.